resources/js/src/app/services/ValidationService.js

```diff
@@ -200,6 +204,42 @@ function _hasValue($formControl)
     return $.trim($formControl.val()).length > 0;
 }
 
+/**
+ * @param {object} $formControl - Input inside Formular
+ * @returns value is valid date
+ */
+function _isValidDate($formControl)
+{
+    var string = $formControl.val();
+    var match = string.match(/^(?:(\d{1,2})[.\/-](\d{1,2})[.\/-](\d{4}))|(?:(\d{4})[.\/-](\d{1,2})[.\/-](\d{1,2}))$/);
+
+    // If match is null date is not valid
+    if (isNull(match))
+    {
+        return false;
+    }
+
+    // Check if year,month,day is valid
+    var currentYear = new Date().getFullYear();
+
+    // YYYY-MM-DD Format
+    if (isUndefined(match[3]))
+    {
+        if ((match[4] >= 1901 && match[4] <= currentYear) && (match[5] >= 1 && match[5] <= 12) && (match[6] >= 1 && match[6] <= 31))
```

Comment:
Ist diese Abfrage verbaut, falls Kunden den Placeholder String in den translations anpassen oder wofür genau?